frontend/app/src/components/inputs/dropdown.tsx

```diff
@@ -27,6 +27,7 @@ import React, { forwardRef, HTMLAttributes, useState } from "react";
 export type DropdownOption = {
   value: string;
   label: string;
+  badge?: string;
```

Comment:
We should never do that. This component is dedicated for dropdown. Here it introduces different behavior and styling based on props.

In this case, since we do that a lot (option with badge), we should create a new component for this specific usecase

Reply:
You could check number pool form for example